src/App.jsx

```diff
@@ -4,41 +4,22 @@ import PropTypes from 'prop-types';
 import '../node_modules/bootstrap-icons/font/bootstrap-icons.css';
```

Comment:
Not wrong, but you could just 
```js
import 'bootstrap-icons/font/bootstrap-icons.css';
```

When you do an import or require, nodejs would first look through the node_modules, if it didn't find anything there, it would look relative to the current directory the file lives in, if it's not there, then error.